docs/src/featureToggle.js

```diff
@@ -2,6 +2,6 @@
 module.exports = {
   enable_website_banner: false,
   enable_toc_banner: true,
-  enable_docsnav_banner: false,
+  enable_docsnav_banner: true,
```

Comment:
This introduces a visual regression:

<img width="793" alt="SCR-20241202-nmwv" src="https://github.com/user-attachments/assets/e691d002-453d-4f94-9490-c6460ec31ffc">

https://mui.com/material-ui/getting-started/